public/app/plugins/datasource/elasticsearch/datasource.ts

```diff
@@ -778,11 +780,17 @@ export class ElasticDatasource extends DataSourceApi<ElasticsearchQuery, Elastic
   }
 
   getMultiSearchUrl() {
+    const searchParams = new URLSearchParams();
+
     if (gte(this.esVersion, '7.0.0') && this.maxConcurrentShardRequests) {
-      return `_msearch?max_concurrent_shard_requests=${this.maxConcurrentShardRequests}`;
+      searchParams.append('max_concurrent_shard_requests', '' + this.maxConcurrentShardRequests);
+    }
+
+    if (gte(this.esVersion, '6.6.0') && this.xpack && this.includeFrozen) {
+      searchParams.append('ignore_throttled', 'false');
     }
 
-    return '_msearch';
+    return (`_msearch?` + searchParams.toString()).replace(/\?+$/, '');
```

Comment:
is the `+` in the regex necessary? my understanding is that you want to remove the `?` from the end when it is at the end (meaning no query-params), so `/\?$/` should be enough, but maybe i'm missing something?

Reply:
i usually do not use backticks-around-strings when i do not need to add variables to the string, so i recommend making these normal single-quotes, but this is just nitpicking, feel free to ignore it 😄

Reply:
you are very much right on both points